.github/workflows/build.yml

```diff
@@ -256,15 +253,34 @@ jobs:
           vcpkgGitCommitId: ${{ env.VCPKG_VERSION }}
           vcpkgTriplet: ${{ matrix.config.vcpkg_triplet }}
 
-      - name: CMake
+      - name: Checkout IrrlichtMT
+        uses: actions/checkout@v2
+        with:
+          repository: minetest/irrlicht
+          path: lib/irrlichtmt/
+
+      - name: IrrlichtMT CMake
+        run: |
+            cmake ${{matrix.config.generator}}  `
+            -DCMAKE_TOOLCHAIN_FILE="${{ github.workspace }}\vcpkg\scripts\buildsystems\vcpkg.cmake"  `
+            -DCMAKE_BUILD_TYPE=Release  `
+            lib/irrlichtmt/ -S lib/irrlichtmt/ -B lib/irrlichtmt/
+
+      - name: Build IrrlichtMT
+        run: cmake --build lib/irrlichtmt/ --config Release -S lib/irrlichtmt/ -B lib/irrlichtmt/
+
+      - name: Minetest CMake
         run: |
           cmake ${{matrix.config.generator}}  `
           -DCMAKE_TOOLCHAIN_FILE="${{ github.workspace }}\vcpkg\scripts\buildsystems\vcpkg.cmake"  `
           -DCMAKE_BUILD_TYPE=Release  `
           -DENABLE_POSTGRESQL=OFF  `
+          -DIRRLICHT_LIBRARY=D:\a\minetest\minetest\lib\irrlichtmt\lib\Win32-gcc\Release\IrrlichtMt.lib  `
```

Comment:
this should use `${{ github.workspace }}`

Reply:
Changed and it seems to work, the docs I found on that weren't crystal clear